src/util/system.cpp

```diff
@@ -1258,16 +1263,6 @@ void SetupEnvironment()
     // Set the default input/output charset is utf-8
     SetConsoleCP(CP_UTF8);
     SetConsoleOutputCP(CP_UTF8);
-#endif
-    // The path locale is lazy initialized and to avoid deinitialization errors
-    // in multithreading environments, it is set explicitly by the main thread.
-    // A dummy locale is used to extract the internal default locale, used by
-    // fs::path, which is then used to explicitly imbue the path.
-    std::locale loc = fs::path::imbue(std::locale::classic());
-#ifndef WIN32
-    fs::path::imbue(loc);
-#else
-    fs::path::imbue(std::locale(loc, new std::codecvt_utf8_utf16<wchar_t>()));
```

Comment:
In commit "refactor: replace boost::filesystem with std::filesystem" (7810d151f5219e634b431c5393771616c759aef7)

Mentioned in a previous comment but I think a better workaround for lack of `path::imbue` than current approach of calling `fs::u8path` manually would be to use namespace solution from https://stackoverflow.com/questions/42765168/change-narrow-string-encoding-or-missing-stdfilesystempathimbue to call it automatically on windows (and not assume any encoding on other platforms)